cli/smartcontract/smart_contract.go

```diff
@@ -98,12 +103,14 @@ func NewCommands() []cli.Command {
 		walletFlag,
 		addressFlag,
 		gasFlag,
+		outFlag,
 	}
 	deployFlags = append(deployFlags, options.RPC...)
 	invokeFunctionFlags := []cli.Flag{
 		walletFlag,
 		addressFlag,
 		gasFlag,
+		outFlag,
```

Comment:
Wouldn't it be useful to add the same flag to `testInvokeFunction`?

Reply:
Actually `testInvokeFunction` may be the only place we may add this flag to, as it doesn't send tx.

Reply:
Turns out `testinvokefunction` doesn't accept wallet and address. I believe it is ok to have this just in `invokeFunction`, to not overload `testInvokeFunction`.

Reply:
Test invocation doesn't need to save anything to file, it can specify whatever signers it wants to.